pages/common/mamba.md

```diff
@@ -1,37 +1,37 @@
 # mamba
 
-> The fast, cross-platform package manager, intended as a drop-in replacement for conda.
+> Fast, cross-platform package manager, intended as a drop-in replacement for conda.
 > Some subcommands such as `mamba repoquery` have their own usage documentation.
 > More information: <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html>.
 
 - Create a new environment, installing the specified packages into it:
 
 `mamba create --name {{environment_name}} {{python=3.10 matplotlib}}`
 
-- List all environments:
+- Update all packages in environment:
 
-`mamba info --envs`
+`mamba update --all`
 
-- Activate an environment:
```

Comment:
I think we could add deactivate back if there's not already 8 commands, sorry for the confusion

Reply:
There are already 8 commands now, so leave as is?